zion/src/main/scala/edu/uci/ics/cloudberry/zion/common/Config.scala

```diff
@@ -31,5 +34,36 @@ object Config {
     FiniteDuration(split(0).toLong, split(1))
   }
 
+  object MemorySize extends Enumeration {
+    val KB: Int = 1024
+    val MB: Int = 1024*1024
+    val GB: Int = 1024*1024*1024
+  }
+
+  def parseFrameLengthLimit(memoryString: String): Int = {
+    val size = memoryString.substring(memoryString.length-2)
```

Comment:
there should be the test codes reflecting what you said above. 
[here](https://github.com/ISG-ICS/cloudberry/blob/65872b8fc21468cf612e2be1323c6a8d922f2894/zion/src/test/scala/edu/uci/ics/cloudberry/zion/common/ConfigTest.scala) is one example.